plugins/outputs/quix/quix.go

```diff
@@ -109,13 +109,18 @@ func (q *Quix) Connect() error {
 			return fmt.Errorf("unsupported SASL mechanism: %s", quixConfig.SaslMechanism)
 		}
 
-		// Certificate
-		certPool := x509.NewCertPool()
-		if !certPool.AppendCertsFromPEM(quixConfig.cert) {
-			return errors.New("appending CA cert to pool failed")
+		// Certificate (optional)
+		if len(quixConfig.cert) > 0 {
+			certPool := x509.NewCertPool()
+			if !certPool.AppendCertsFromPEM(quixConfig.cert) {
+				return errors.New("appending CA cert to pool failed")
+			}
+			cfg.Net.TLS.Enable = true
+			cfg.Net.TLS.Config = &tls.Config{RootCAs: certPool}
+		} else {
+			cfg.Net.TLS.Enable = true
+			cfg.Net.TLS.Config = &tls.Config{InsecureSkipVerify: true}
```

Comment:
No, we cannot simply enforce unsecure settings on our users. If you need to allow insecure settings, make this explicit by adding a setting! Why would we need to set this at all?

Reply:
You are right. It is not needed.